app/scripts/background.js

```diff
@@ -345,12 +347,22 @@ function setupController(initState, initLangCode) {
    */
   function connectRemote(remotePort) {
     const processName = remotePort.name;
-    const isMetaMaskInternalProcess = metamaskInternalProcessHash[processName];
 
     if (metamaskBlockedPorts.includes(remotePort.name)) {
       return;
     }
 
+    let isMetaMaskInternalProcess = false;
+    const sourcePlatform = getPlatform();
+
+    if (sourcePlatform === PLATFORM_FIREFOX) {
+      isMetaMaskInternalProcess = metamaskInternalProcessHash[processName];
+    } else {
```

Comment:
Are there other `sourcePlatforms` that are not `PLATFORM_FIREFOX` for which this will still work? Opera? Brave? cc @jpuri

Reply:
Other browsers are chromium based so should work well. But good point, I can test it once to confirm.

Reply:
Yea just want to make sure that this format is the same across them.

Reply:
I tested it on brave and it works as expected.

Reply:
Should we test on Opera too since it would be pretty consequential for those users if for some reason this didn't work?

Reply:
Yep just tested in opera also and works as expected.